operations/policies/deployment_rollingupdate.rego

```diff
@@ -0,0 +1,49 @@
+package main
+
+import future.keywords.in
+
+ignored_deployments = [
+	"continuous-test", # We only want to run at most one instance at a time.
+	"rollout-operator", # We only want to run at most one instance at a time.
```

Comment:
We don't run the rego policies on subcharts, so this won't have any effect. I think we should remove this from the exclusions.

Reply:
This policy is also applied to the rendered Jsonnet manifests, which does include `rollout-operator`.